_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+
+The traffic manager now exposes the `get_next_action` function to access the next action that a traffic manager guided vehicle will take and the `get_all_actions` enables the user to query all possible actions a traffic manager guided vehicle has available.
+
+The `set_path` function allows a custom route to be defined using coordinates defined as CARLA Locations:
+
+```py
+path = [carla.Location(x=-506.696198, y=179.384308, z=0.038194),
+                    carla.Location(x=-504.745972, y=232.868927, z=0.039417)]
+traffic_manager.set_path(vehicle, path)
+```
+The `set_route` function allows a custom route to be defined using route commands:
+
+```py
+route = ["Right", "Straight", "Right", "Right", "Left", "Right"]
+ traffic_manager.set_route(vehicle, route)
+```
+
+![set_route](/img/posts/2021-11-07/follow_route.gif){:class="img-fluid rounded mx-auto d-block"}
+
+### Traffic manager controlled vehicle lights
+
+The traffic manager now has functionality to control vehicle lights. Headlights, fog lights and indicators are controlled according to vehicle route data and weather conditions. Indicator lights will activate according to the next planned turn in the route, braking lights will activate when the vehicle brakes are applied and fog lights and main beams are activated according to the weather and light conditions. During night, the vehicle main beams are activated and in foggy conditions the rear fog lights are activated. 
+
+
+
+### Improved vehicle behavior - gears and braking
+
+Vehicle physics has been improved with updated acceleration, braking and gear behaviors. The acceleration and braking behavior of the vehicles has been carefully remodelled to more accurately capture the behavior of the CARLA garage vehicles’ real world counterparts, with much care taken to reproduce published braking distance behavior as accurately as possible. Similarly the gear change behavior has been altered to better model real world behavior gear changes.
+
+![accelleration](/img/posts/2021-11-07/acceleration.gif){:class="img-fluid rounded mx-auto d-block"}
+
+
+## Python library upgrades
+
+Continuing to improve the useability of the CARLA Python API, the CARLA agents package that allows custom control of NPC vehicles within the simulation is now distributed through PyPi. Up until now the CARLA agents package has been delivered as an egg file in the CARLA core package, requiring environment variable adjustments to function .`carla-agents` is now available as a python wheel through PyPi, allowing a convenient installation through the native python package manager PIP.
+
+![pip_install](/img/posts/2021-11-07/pip_install_carla_agents.gif){:class="img-fluid rounded mx-auto d-block"}
+
+
+New wheels are now available for Python version 3.9 for both Linux and Windows. 
+
+
+
+## Changelog 
```

Comment:
Just throwing it out there, we should have a standarized CHANGELOG format

Reply:
true I'll talk to Bernat about it